### ISSUE TITLE
Can't make it work when I setup contentDir configuration

### ISSUE THREAD

**Issue by oliviernguyenquoc** [seanlane/gochowdown#23]
Hi,

I was wondering whether this theme support ["contentDir"](https://gohugo.io/getting-started/configuration/) option in Hugo configuration file, or better [Module config mounts](https://gohugo.io/hugo-modules/configuration/#module-config-mounts).

I would like to use an external folder to version my recipes with my other markdowns.

Maybe this is not due to the theme but a miss-configuration on my side. What your guess on that ?

Thanks for your project. This is really cool.

**Comment by seanlane** [comment]
It likely should, but it's not something that I have ever tested so there it's possible that the theme isn't written correctly to use it. I'll try testing this scenario when I have a moment to do so.

**Comment by oliviernguyenquoc** [comment]
Thanks !
If you have a single hint on how to fix that, please let me know.

**Comment by seanlane** [comment]
Yeah, looks like it does support this, it's not theme dependent. Right now, there's no explicit option in the `config.yml` of the example site, but the default is just `content`. To change that, you can either run hugo with the `contentDir` flag: `hugo --contentDir /path/to/your/content` or set the option within the `config.yml` file like this

```
baseURL: https://example.com
disablePathToLower: true
languageCode: en-us
contentDir: /path/to/your/content
title: My Recipes
theme: gochowdown
...
```
